lib/core/jdl_application_configuration_option.js

```diff
@@ -28,11 +28,19 @@ module.exports = class JDLApplicationConfigurationOption {
     this.value = value;
   }
 
+  getName() {
+    return this.name;
+  }
+
   getValue() {
     return this.value;
   }
 
   toString() {
-    return `${this.name} ${this.value}`;
+    return this._toString(this.value);
+  }
+
+  _toString(value) {
+    return `${this.name} ${value}`;
```

Comment:
why have two methods?
if the second one is to be overridden, might as well have one method and let subclasses override the one